manifests/c/CyberSift/TutelaWindowsAgent/1.5.0/CyberSift.TutelaWindowsAgent.locale.en-US.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: CyberSift.TutelaWindowsAgent
+PackageVersion: 1.5.0
+PackageLocale: en-US
+Publisher: CyberSift
+PackageName: Tutela Windows Agent
+License: Commercial
+ShortDescription: Tutela Windows Agent

```

Comment:
Can you please provide a better description of what this software is or what it is used for?

Reply:
Yes sure!
Updated short description with: `Tutela Windows Agent. This agent enumerates software running on a workstation to check for applicable CVEs. It also runs several CIS benchamrk checks. (https://www.cybersift.com/tutela)`